src/store/app.js

```diff
@@ -56,4 +72,6 @@ class AppStore {
 
 const store = new AppStore();
 
+store.switchLanguage(localStore.get('locale') || 'en');
```

Comment:
Can we pull the fallback language (`en`) from `i18n` to avoid the hard code here?